README.md

```diff
@@ -41,6 +163,47 @@ puts config.paths   # "a", "b"
 puts config.ignore  # "c"
 ```
 
+List all files typchecked by Sorbet:
+
+```ruby
+config = Spoom::Sorbet::Config.parse_file("sorbet/config")
+puts Spoom::Sorbet.srb_files(config)
+```
+
+#### Parsing Sorbet metrics
+
+Display metrics collected during typechecking:
+
+```ruby
+puts Spoom::Sorbet.srb_metrics(capture_err: false)
+```
+
+#### Interacting with LSP
+
+Create a LSP client:
```

Comment:
```suggestion
Create an LSP client:
```

Reply:
Dang, I hesitated

Reply:
It really depends on how you expect it to be read. If you expect people to say _El-Es-Pee_ then `an` is appropriate. If it will be read as _Language Server Protocol_ then `a` is fine.